docs/sources/tempo/setup/set-up-test-app.md

```diff
@@ -123,45 +124,40 @@ To fix the error, [upgrade your Grafana to 9.0 or later](/docs/grafana/latest/se
 Once you have created a data source, you can visualize your traces in the **Grafana Explore** page.
 For more information, refer to [Tempo in Grafana]({{< relref "../getting-started/tempo-in-grafana" >}}).
 
-### Test your configuration using the TNS application
+### Use OpenTelemetry `telemetrygen` to generate tracing data
```

Comment:
any particular reason telemetrygen? and not xk6-client-tracing?

Reply:
I think part of the reasoning was the ease of creating testing data. We could add another option to the page for K6-tracing. What do you think?  

The K6-tracing Docker image is used as the test app in the Linux set up guide.

Reply:
It feels way easier to install a component that's part of OTel to me rather than have to potentially go install Docker and then create a compose manifest to do this. I'm ambivalent either way, but `telemtrygen` has become the way CS quickly test whether Tempo/GET/GCT is working and it just feels a far lower lift.

If we've strong feelings against it, we can change it. :)